install/boards/bcm_28xx.sh

```diff
@@ -28,4 +28,7 @@ done
 
 # Remove any console serial configuration
 echo "- Configure serial."
-sudo sed -e 's/console=serial[0-9],[0-9]*\ //' -i /boot/cmdline.txt
\ No newline at end of file
+sudo sed -e 's/console=serial[0-9],[0-9]*\ //' -i /boot/cmdline.txt
+
+echo "- Run rpi update."
+sudo rpi-update
```

Comment:
Since this is raspbian only, we should check the OS for that.
In this case, you can load `/etc/os-release` and compare `ID == raspbian`

Be also aware that rpi-update will update the system without an specific version in mind, but using the latest version available under the master branch of the raspberry project.

> WARNING: Pre-release versions of software are not guaranteed to work. You should not use rpi-update on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process.

> rpi-update will download the latest pre-release version of the linux kernel, its matching modules, device tree files, along with the latest versions of the VideoCore firmware. It will then install these files to relevant locations on the SD card, overwriting any previous versions.

> All the source data used by rpi-update comes from the GitHub repo https://github.com/Hexxeh/rpi-firmware. This repository simply contains a subset of the data from the official firmware repository, as not all the data from that repo is required.

ref: https://www.raspberrypi.org/documentation/raspbian/applications/rpi-update.md